README.md

```diff
@@ -8,6 +8,10 @@ Idiomatic Rust client libraries for [Google Cloud Platform](https://cloud.google
 > **NOTE:** this project is under development, all APIs are subject to change
 > without notice. Some documentation is aspirational.
 
+## Getting Started
+
+The [User Guide] includes basic tutorials to get
```

Comment:
"... to get familiar with the Google Cloud Rust SDK."

?

Reply:
Fixed, thanks.